### ISSUE TITLE
CORE - Help FAQs

### ISSUE THREAD

**Issue by codingwithmanny** [berachain/docs-old#364]
**Context:**
Review this section to make sure that all wording matches the latest information.

**Page:**
https://docs.berachain.com/learn/help/faqs

**Tasks:**
1. Verify information

**Comment by camiinthisthang** [comment]
@DonBera https://berachain-docs-core-next-git-contentchecks.berachain-staging.com/learn/help/faqs

**Comment by camiinthisthang** [comment]
<img width="1266" alt="Image" src="https://github.com/user-attachments/assets/cf07744a-9346-455d-afd1-a46f5ff2a1fc" />

no comments left here, please review here https://berachain-docs-core-next.berachain-staging.com/learn/help/faqs

**Comment by DonBera** [comment]
but you addressed the previous comment or not showing any of that? Leaving some other minor comment

**Comment by camiinthisthang** [comment]
yes the previous comments were addressed, and just addressed this recent one

**Comment by DonBera** [comment]
all good for me
